src/components/Navbar/Navbar.style.tsx

```diff
@@ -15,6 +15,7 @@ export const NavbarContainer = styled(AppBar)<{ sticky?: boolean }>(
     justifyContent: 'space-between',
     position: 'sticky',
     top: 0,
+    minWidth: 416,
```

Comment:
are we sure about this @dennyscode ?

Reply:
Good find, removed it as this is done in another PR anyways